docs/decision_records/cli/CLI-002-self-hosted-init-and-uninstall-behaviors.md

```diff
@@ -4,28 +4,29 @@
+  * Create the default `components` folder in `$HOME/.dapr` for Linux/MacOS or `%USERPROFILE\.dapr` for Windows.
+  * Create the default components configurations for `pubsub.yaml`, `statestore.yaml` and `zipkin.yaml` in the default `components` folder.
   * Create a default configuration file in `$HOME/.dapr/config.yaml` for Linx/MacOS and `%USERPROFILE%\.dapr\config.yaml` for Windows for enabling tracing by default.
 * Calling `dapr init --slim` will
-  * Install the binaries `daprd` and `placement` in `/usr/local/bin` for Linux/MacOS and `C:\dapr` for Windows.
-  * Create the empty default components folder in `$HOME/.dapr` for Linux/MacOS or `%USERPROFILE\.dapr` for Windows.
-* CLI on the init command will fail if a prior installtion exists in the default paths `/usr/local/bin` for Linux/MacOS or `C:\dapr` for Windows or the provided install path using the `--install-path` flag.
+  * Install the binaries `daprd` and `placement` in `$HOME/.dapr` for Linux/MacOS and `%USERPROFILE%\.dapr` for Windows.
+  * Create an empty default `components` folder in `$HOME/.dapr` for Linux/MacOS or `%USERPROFILE\.dapr` for Windows.
 * Calling `dapr uninstall` will
-  * Remove the binary daprd (and placement for slim init) from the default path `/usr/local/bin` for Linux/MacOS and `C:\dapr` for Windows. The path can be overridden with a `--install-path` flag.
+  * Remove the binary daprd (and placement for slim init) from the default path `$HOME/.dapr` for Linux/MacOS and `%USERPROFILE%\.dapr` for Windows.
   * Remove the docker dapr_placement if Docker is installed.
 * Calling `dapr uninstall --all`
-  * * Remove the binary daprd (and placement for slim init) from the default path `/usr/local/bin` for Linux/MacOS and `C:\dapr` for Windows. The path can be overridden with a `--install-path` flag.
   * Remove the docker containers dapr_placement, dapr_redis and dapr_zipkin if Docker is installed.
   * Remove the default folder `$HOME/.dapr` in Linux/MacOS and `%USERPROFILE%\.dapr` in Windows.
+* CLI on the init command will fail if a prior installtion exists in the default path `$HOME/.dapr` for Linux/MacOS and `%USERPROFILE%\.dapr` for Windows.
+* **There will no longer be an option for `--install-path` during init or during uninstall.**
+* The `dapr` CLI by default will expect the `daprd` in `$HOME/.dapr` for Linux/MacOS and `%USERPROFILE%\.dapr` for Windows. The command `dapr run` will not expect the `daprd` binary to be in the `PATH` variable, it will launch the binary from the default path.
   
 ## Consequences
 
-This will require Linux users, running in standalone mode  to use `sudo dapr init` for initialization(existing behavior) and also run `sudo dapr uninstall` for uninstall calls.
+Only the `dapr` binary will be installed as it has been done previously. All other binaries and configurations that dapr needs (on running `dapr init`)will be placed in the path  `$HOME/.dapr/bin` for Linux/MacOS and `%USERPROFILE%\.dapr` for Windows.
```

Comment:
Can we remove the first line as this is describing the consequence of `dapr init`